### DISCUSSION TITLE
Schedule Trigger errors

### DISCUSSION THREAD

**Discussion by evansexon007** [microsoft/finops-toolkit#871] [General]
Hi Everyone, new deployment of 0.4

My schedule triggers are not started and when I try to start them I get the following error

![image](https://github.com/user-attachments/assets/ffc3f900-2d2c-4340-b51e-8b92ec98792d)

![image](https://github.com/user-attachments/assets/ab1b7abc-d778-47b2-9fad-5b1b00884f0c)

Can you please help suggest what I need to do to fix this?

Thank you

**Comment by flanakin** [comment]
This is a known bug, which [we've fixed](https://github.com/microsoft/finops-toolkit/pull/843) and will ship with the next update in a week or two. Here's how you can fix it:

1. Open the `config_ConfigureExports` pipeline
2. Find and open the `Create or update open month focus export` activity
3. In the web request body, find and change "TheLastMonth" to "MonthToDate"
4. Then find and open the `Create or update closed month focus export` activity
5. In the web request body, find and change "MonthToDate" to "TheLastMonth"

These got inverted in the template.

**Reply by evansexon007** [reply]
Hi Thanks for you reply, I have already done that change previously as I read in a previous discussion 

Create or update open month focus export

{ "properties": { "definition": { "dataSet": { "configuration": { "dataVersion": "1.0", "filters": [] }, "granularity": "Daily" }, "timeframe": "MonthToDate", "type": "FocusCost" }, "deliveryInfo": { "destination": { "container": "msexports", "rootFolderPath": "providers/Microsoft.Billing/billingAccounts/xxxx/departments/xxxx", "type": "AzureBlob", "resourceId": "@{variables('storageAccountId')}" } }, "schedule": { "recurrence": "Daily", "recurrencePeriod": { "from": "2024-08-02T00:00:00.000Z", "to": "2050-02-01T00:00:00.000Z" }, "status": "Inactive" }, "format": "Csv", "partitionData": true, "dataOverwriteBehavior": "CreateNewReport", "compressionMode": "None" }, "id": "@{variables('resourceManagementUri')}@{item().scope}/providers/Microsoft.CostManagement/exports/@{variables('exportName')}", "name": "@{variables('exportName')}", "type": "Microsoft.CostManagement/reports", "identity": { "type": "systemAssigned" }, "location": "global" }

Create or update closed month focus export


{ "properties": { "definition": { "dataSet": { "configuration": { "dataVersion": "1.0", "filters": [] }, "granularity": "Daily" }, "timeframe": "TheLastMonth", "type": "FocusCost" }, "deliveryInfo": { "destination": { "container": "msexports", "rootFolderPath": "providers/Microsoft.Billing/billingAccounts/xxxx/departments/xxxx", "type": "AzureBlob", "resourceId": "@{variables('storageAccountId')}" } }, "schedule": { "recurrence": "Monthly", "recurrencePeriod": { "from": "2024-08-02T00:00:00.000Z", "to": "2050-02-01T00:00:00.000Z" }, "status": "Inactive" }, "format": "Csv", "partitionData": true, "dataOverwriteBehavior": "CreateNewReport", "compressionMode": "None" }, "id": "@{variables('resourceManagementUri')}@{item().scope}/providers/Microsoft.CostManagement/exports/@{variables('exportName')}", "name": "@{variables('exportName')}", "type": "Microsoft.CostManagement/reports", "identity": { "type": "systemAssigned" }, "location": "global" }

config_ConfigureExports pipeline is running without issues

![image](https://github.com/user-attachments/assets/ba003cda-ec70-468d-9fca-e45dc7f793eb)

The issue seems to be with the scheduled triggers which are in a stopped state and can't be started

**Reply by evansexon007** [reply]
Can you please help me 

What is meant to go in the Body

![image](https://github.com/user-attachments/assets/18169781-f339-488b-af24-18d04affef50)

**Comment by evansexon007** [comment]
My triggers are now started.
